__tests__/server.test.js

```diff
@@ -0,0 +1,41 @@
+const app = require(`../server/app`);
+const db = require('../db/connection');
+const data = require('../db/data/test-data');
+const seed = require('../db/seeds/seed');
+const request = require('supertest');
+
+beforeEach(() => {
+    return seed(data)
+});
+
+afterAll(() => {
+    return db.end()
+});
+
+describe('general errors', () => {
+    test('404: should return a 404 if the endpoint does not exist', () => {
```

Comment:
The reason for testing this would be to add your own "personalised" 404 message. Without asserting for this message I think this test would have passed straight away? This is due to Express handling returning 404 status codes. See the error handling notes for more clarification.